charts/node-local-dns/templates/podmonitor.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "node-local-dns.fullname" . }}
+  labels:
+    {{- include "node-local-dns.labels" . | nindent 4 }}
+    {{ toYaml .Values.podmonitor.promOperatorSelector | nindent 4 }}
+  annotations:
+  {{- with .Values.podmonitor.annotations }}
```

Comment:
@wonderg **with** action should be before `annotations:`

Reply:
Agree, just fixed it. Please check